.travis.yml

```diff
@@ -0,0 +1,27 @@
+sudo: required
+
+language: bash
+
+services:
+  - docker
+
+install:
+  - docker build -t quay.io/wikiwatershed/rwd:${TRAVIS_COMMIT:0:7} .
+
+script:
+  - docker run -d --name rwd quay.io/wikiwatershed/rwd:${TRAVIS_COMMIT:0:7}
```

Comment:
Hmm. Maybe not. Here's the output from the travis log:

```
The command "docker run -d --name rwd quay.io/wikiwatershed/rwd:${TRAVIS_COMMIT:0:7}" exited with 0.
```

Probably copied it wrong or it's not needed.